MATLAB/Utilities/IO/VarianCBCT/HUMapping.m

```diff
@@ -0,0 +1,27 @@
+function HU3D = HUMapping(img3D, Protocol)
+% Pixel value to HU value Mapping
+% Method: using CATPhan 604 calibration data
+% Input:
+%       img3D: reconstructed image matrix in pixel value
+%       Protocol: CBCT scan protocol
+% Output:
+%       HU3D: image matrix in CT HU value
+% Date: 2021-09-03
+% Author: Yi Du, yi.du@hotmail.com
+
+HU3D = zeros(numel(img3D), 1);
+load('D:\MATLABWorkplace\Ongoing_Projects\Fronc_VarianCBCT\demo_Pixel2HU.mat', 'Pixel2HU');
```

Comment:
This may need to change. Either add the file to TIGRE, or we need some error/warning + explanation on what this file is.

Reply:
Oh, yes. The file is attached then. 
There should be warning that users are recommended to make their own Pixel2HU calibration file.
[demo_Pixel2HU.zip](https://github.com/CERN/TIGRE/files/8746944/demo_Pixel2HU.zip)